test/e2e/scenarios/test.js

```diff
@@ -0,0 +1,31 @@
+describe('Google demo test for Mocha', function () {
+  describe('with Nightwatch', function () {
+    before(function (client, done) {
+      done()
+    })
+
+    after(function (client, done) {
+      client.end(function () {
+        done()
+      })
+    })
+
+    afterEach(function (client, done) {
+      done()
+    })
+
+    beforeEach(function (client, done) {
+      done()
+    })
+
+    it('uses TDD to run the Google simple test', function (client) {
```

Comment:
Can we remove the google test and replace it with a test against the specification you added at `1.json`?